<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>g-SEGS</string>
  </property>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>560</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>491</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Service</string>
   </property>
   <widget class="QLineEdit" name="serviceLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>441</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="checkServiceButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>21</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>491</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Task</string>
   </property>
   <widget class="QComboBox" name="classValComboBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Target attribute value:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rankedRadioButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ranked data</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="labelledRadioButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Labelled data</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>251</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Search constraints</string>
   </property>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Min subgroup size</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Max number of terms</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Max number of subgroups</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="minSizeSGSpinBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="maxNumTermsSpinBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="maxNumSGSpinBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>99</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>48</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Cutoff</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="cutoffSpinBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>300</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>231</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Output parameters</string>
   </property>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Random seed</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="randomSeedSpinBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Max times covered (wracc)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="kSpinBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="summarizeCheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Summarize descriptions</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>420</y>
     <width>491</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Legacy settings</string>
   </property>
   <widget class="QCheckBox" name="legacyCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Use legacy SEGS ontology</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Combine p-values:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>48</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Fisher</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="fisherSpinBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>62</width>
      <height>31</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="gseaSpinBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>62</width>
      <height>31</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>48</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>GSEA</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="pageSpinBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>62</width>
      <height>31</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>48</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>PAGE</string>
    </property>
   </widget>
   <widget class="QComboBox" name="pValueComboBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>78</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Maximal p-value</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>340</y>
     <width>491</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>General concept levels</string>
   </property>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ont 1</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="level_ont1SpinBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="level_ont2SpinBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ont 2</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="level_ont3SpinBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ont 3</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="level_ont4SpinBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ont 4</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
